.github/workflows/ubuntu_stack_vols.yml

```diff
@@ -0,0 +1,223 @@
+name: Ubuntu_stack_vols
+
+on:
+  push:
+    branches: [ master, dev ]
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - 'docs/**'
+      - 'case_studies/**'
+  pull_request:
+    branches: [ master, dev ]
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - 'case_studies/**'
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+        - name: Set up dependencies
+          run: |
+            sudo apt-get update
+            sudo apt-get -y install automake autoconf libtool libtool-bin m4 cmake
+            # The MPICH installed on github actions is too slow
+            # sudo apt-get install mpich
+            # mpicc -v
+            # zlib
+            sudo apt-get -y install zlib1g-dev
+        - name: Build MPICH
+          if: ${{ success() }}
+          run: |
+            WORKDIR=$(pwd)
```

Comment:
Instead of using env pwd, use the one formally supported by github, GITHUB_WORKSPACE
See an example in https://github.com/Parallel-NetCDF/E3SM-IO/blob/master/.github/workflows/ubuntu_ompi.yml